src/constants.js

```diff
@@ -0,0 +1,3 @@
+const cohortCapacity = 24
```

Comment:
Interesting you kept this is its own file. Can you elaborate on why this isn't just stored above your branch or organization class?

Reply:
At this point it's a bit overdoing it but I like having a constants.js file to manage e.g. authentication tokens, etc. for the general application. Think that part of my brain decided on this, even though you are of course correct in saying it would make sense in the cohort.js file, too